packages/server/lib/cloud/studio/StudioElectron.ts

```diff
@@ -0,0 +1,50 @@
+// tslint:disable-next-line no-implicit-dependencies - electron dep needs to be defined
+import { BrowserWindow } from 'electron'
+import Debug from 'debug'
+
+const debug = Debug('cypress:server:studio:electron')
+
+export class StudioElectron {
+  private browserWindow: BrowserWindow | undefined
+
+  async createBrowserWindow () {
```

Comment:
Does this need to be async?

Reply:
Hmm good catch, not anymore. Used to do some fixture loading that was awaited here. Will remove and update the typings.